drainer/sync/syncer.go

```diff
@@ -24,6 +24,9 @@ type Item struct {
 	PrewriteValue *pb.PrewriteValue // only for DML
 	Schema        string
 	Table         string
+
+	// the finish TS executed in downstream, only for tidb
+	FinishTS int64
```

Comment:
is `AppliedTS` better?

Reply:
updated https://github.com/pingcap/tidb-binlog/pull/670/commits/4c8782f0e021d79d1b1ecb9094716dc0a9810c2f